analysers/Analyser_Osmosis.py

```diff
@@ -204,6 +204,34 @@ class Analyser_Osmosis(Analyser):
 CREATE INDEX idx_multipolygons_poly_proj ON {0}.multipolygons USING gist(poly_proj);
 CREATE INDEX idx_multipolygons_tags ON {0}.multipolygons USING gist (tags);
 ANALYZE {0}.multipolygons;
+"""
+
+    sql_create_polygons = """
+CREATE UNLOGGED TABLE {0}.polygons AS
+SELECT
+    'R' AS type,
+    *
+FROM
+    {0}.multipolygons
+UNION ALL
+SELECT
+    'W' AS type,
+    id,
+    tags,
+    CASE WHEN ST_IsValid(linestring) = 't' AND ST_IsSimple(linestring) = 't' AND ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(linestring) ELSE NULL END AS poly,
```

Comment:
```suggestion
    ST_MakePolygon(linestring) AS poly,
```
the checks for validity of the linestring are already included in the definition of `is_polygon`
https://github.com/osm-fr/osmose-backend/blob/dev/osmosis/ImportDatabase_Prepare.sql#L15-L17
(I suspect the ST_Transform was unintended here; they can differ, see https://github.com/osm-fr/osmose-backend/issues/2058#issuecomment-1788769091)